run_snafu.py

```diff
@@ -108,7 +109,7 @@ def process_generator(index_args, parser):
         for data_object in wrapper_object.run():
             for action, index in data_object.emit_actions():
 
-                es_index = index_args.prefix + index
+                es_index = index_args.prefix + '-' + index
```

Comment:
This will require all preexisting wrappers that have been integrated into run_snafu to have the leading dash removed or all indices will be broken. Do not merge until this is accounted for.

Reply:
I did account for it, there are only 3 benchmarks using run_snafu at this time, and they are all in the PR.
```
[bengland@bene-laptop ripsaw]$ grep -r run_snafu.py .
./roles/fio-distributed/templates/client.yaml:             cat /tmp/fio/fiojob-{{job}}-{{bs}}-{{numjobs}}; mkdir -p /tmp/fiod-{{ uuid }}/fiojob-{{job}}-{{bs}}-{{numjobs}}; python /opt/snafu/run_snafu.py -t fio -H /tmp/host/hosts -j /tmp/fio/fiojob-{{job}}-{{bs}}-{{numjobs}} -s {{fiod.samples}} -d /tmp/fiod-{{ uuid }}/fiojob-{{job}}-{{bs}}-{{numjobs}};
./roles/fs-drift/templates/workload_job.yml.j2:              python run_snafu.py
./roles/smallfile-bench/templates/workload_job.yml.j2:              python /opt/snafu/run_snafu.py 
```
look in https://github.com/cloud-bulldozer/snafu/pull/99/files